cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -148,11 +148,12 @@ def test_suspend_vm_service_details(context):
     pass
 
 
-@pytest.mark.meta(coverage=[1677744])
-@pytest.mark.manual
+@pytest.mark.meta(automates=[1677744])
+@pytest.mark.customer_scenario
 @pytest.mark.ignore_stream('5.10')
 @pytest.mark.tier(2)
```

Comment:
- Test against **5.10**. I can see fixed clone for 5.10 [here](https://bugzilla.redhat.com/show_bug.cgi?id=1677744#c22)
- Also change **startsin** version